interfaces/Portal/src/app/components/physical-card-popup/physical-card-popup.component.html

```diff
@@ -68,67 +69,77 @@
 
     <!-- Current balance -->
     <ion-item>
-      <ion-label class="card-key-value"
-        >{{
+      <ion-label class="card-key-value">
+        {{
           'registration-details.physical-cards-overview.current-balance'
             | translate
         }}:
       </ion-label>
-      <ion-label class="card-key-value ion-text-right"
-        ><strong
-          >{{ card?.balance / 100 | number: '1.2-2' }} {{ currency }}</strong
-        >
+      <ion-label class="card-key-value ion-text-end">
+        <strong>
+          {{
+            card?.balance / 100 | currency: currency : 'code' : '1.2-2' : locale
+          }}
```

Comment:
The position of the Currency-symbol in relation to the number is not always the same in every language.

This way will fix that for us.